z3c/dependencychecker/dependencychecker.py

```diff
@@ -501,9 +514,9 @@ def imports_from_doctests(path):
     for path, dirs, files in os.walk(path):
         for filename in [os.path.abspath(os.path.join(path, filename))
                          for filename in files
-                         if fnmatch.fnmatch(filename, '*.txt')
-                         or fnmatch.fnmatch(filename, '*.rst')
-                         or fnmatch.fnmatch(filename, '*.py')]:
+                         if fnmatch.fnmatch(filename, '*.txt') or
+                         fnmatch.fnmatch(filename, '*.rst') or
```

Comment:
Yes.  the pep8 checker says: `W503 line break before binary operator`.
Ah, that is a recent change that is added to the pep8 guidelines (but not visible on the official pep yet).
See the links in this issue: https://github.com/PyCQA/pycodestyle/issues/498